CHANGELOG.md

```diff
@@ -23,6 +23,13 @@
   expressions like `filter(where:).isEmpty` instead of `contains(where:)`.  
   [Marcelo Fabri](https://github.com/marcelofabri)
 
+* Add `contains_over_range_not_nil` opt-in rule to prefer using `contains` over comparison of `range(of:)` to `nil`.  
+  [Colton Schlosser](https://github.com/cltnschlosser)
+  [#2776](https://github.com/realm/SwiftLint/issues/2776)
+
+* Make `contains_over_first_not_nil` rule also match `first(where:) == nil`.  
```

Comment:
Maybe we should split this into a different PR and discuss there? I'm a bit conflicted because while I agree that matching `== nil` is useful, the rule is named `contains_over_first_not_nil`.